keras/src/wrappers/random_state.py

```diff
@@ -0,0 +1,73 @@
+import os
+import random
+from contextlib import contextmanager
+from typing import Generator
+from typing import Iterator
+
+import numpy as np
+
+try:
+    import tensorflow as tf
```

Comment:
We should not unconditionally try to import tensorflow since that could cause issues with other backends